src/integrations/receipt.test.ts

```diff
@@ -46,8 +46,6 @@ test("Receipt test", async ({ expect }) => {
   expect(receipts).toBeDefined();
   for (const receipt of receipts) {
     expect(receipt).toBeDefined();
-    expect(receipt.gasPrice).toBeDefined();
```

Comment:
Is it processed recipe?

Reply:
What does it mean?

Reply:
Seems that in processed recipe you always have gasPrice because of `receipt.gasPrice ? BigInt(receipt.gasPrice) : 0n,`